templates/base_index.html

```diff
@@ -25,9 +25,13 @@
     'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
     })(window,document,'script','dataLayer','GTM-M5BPGQ6');</script>
 
+<<<<<<< HEAD
+    <title>{% block title %}Publisher of Ubuntu{% endblock %} | Canonical</title>
+=======
     <!-- Google optomise -->
     <script async src="https://www.googleoptimize.com/optimize.js?id=OPT-KMJNW6P"></script>
     <title>{% block title %}Canonical | Trusted open source for enterprises{% endblock %}</title>
+>>>>>>> a1d2d29f0a01ac81c883ff4dfc6e161cb690682d
```

Comment:
The `<<<<<<< HEAD`, `=======` and `>>>>>>> a1d2d29f0a01ac81c883ff4dfc6e161cb690682d` lines should be removed, as well as the 2 lines that you had originally removed. See suggestion below:
```suggestion
    <title>{% block title %}Canonical | Trusted open source for enterprises{% endblock %}</title>
```